recipes-debian/libevent/libevent_debian.bb

```diff
@@ -36,6 +36,8 @@ DEPENDS = "zlib"
 
 BBCLASSEXTEND = "native nativesdk"
 
+CVE_VERSION = "${@d.getVar('PV').split('-')[0]}"
```

Comment:
Did you find any difference in results between "2.1.8" and "2.1.8-stable" ?

Reply:
There is no difference in results now Because CVE doesn't published to 2.1.8.

Reply:
Okay, your fix seems correct.
The "stable" doesn't appear in version or update attributes. So we should use "2.1.8" style.
https://nvd.nist.gov/vuln/detail/CVE-2015-6525